configuration/addons.md

```diff
@@ -6,14 +6,21 @@ layout: documentation
 
 Add-ons can be installed in different ways, described below.
 
-## Through UI
+## Through User Interface
 
 Log into your admin account (if not already done).
-Navigate to `Settings` and have a look for the add-ons section.
-Search for the desired add-on in the categories and press install.
+Navigate to **Settings** and have a look for the add-ons section.
+Search for the desired add-on in the categories and press 'install'.
 
 ![installing Add-ons through UI](./images/InstallAddonsThroughUi.gif)
 
+Alternatively navigate to **Add-on Store**.
+The openHAB framework will already have scanned your PC and LAN to discover home automation equipment in your system.
+And based on the discovery results it will show the add-ons that it recommends to install.
+Choose the desired add-on and press 'install'.
+
+![image](https://github.com/openhab/openhab-docs/assets/893994/d922f40b-4ace-4400-9378-0e66f2f894c1)
```

Comment:
I have a similar question here https://github.com/openhab/openhab-docs/pull/2178

This indirectly builds a link to AWS to github ... I am not sure if that's good idea.

Until now I have never seen an image in our docs to link with an absolute path .../assets. You can see one example with a relative path at console.html but it only includes css that way.

@Confectrician  Can you give advice how we should handle that?

Reply:
Here is the image

![image](https://github.com/openhab/openhab-docs/assets/893994/3e6ae8ee-d4bf-4a48-ae35-0899deaa41e6)

Reply:
Yep, I looked at the image. It looks pretty nice but it is pointing to github and if we want to linkt to that which is why I wanted to have @Confectrician 's opinion.

Reply:
@stefan-hoehn I posted the image here so that -- whenever you and @Confectrician have sorted it out -- you or he can put it wherever you need it in order for it to build properly.

Reply:
I think I fixed it.